presto-docs/src/main/sphinx/admin/dynamic-filtering.rst

```diff
@@ -192,6 +192,19 @@ especially when a range of values is selected from the build side of the join.
 The limits for min-max filters collection are defined by the properties
 based on ``range-row-limit-per-driver``.
 
+Dimension tables layout
+-----------------------
+
+Dynamic filtering works the best for dimension tables where
+table keys are correlated with columns. For example, for date
+dimension key column should be correlated with date column
+(table keys should monotonically increase with dates).
+For address dimension key column might be composed of other
+columns. For instance it could have format of ``COUNTRY-STATE-ZIP-ADDRESS_ID``
+(e.g `US-NY-10001-1234`). This way dynamic filtering would
```

Comment:
This usage allows dynamic filtering to succeed even with a large number of selected rows from the dimension table.